test/studies/arkouda/sub_test

```diff
@@ -60,63 +74,65 @@ else
   fi
 fi
 
-# Install Arkouda and python dependencies to a python-deps subdir
-export PYTHONUSERBASE=$ARKOUDA_HOME/python-deps
-# If Arkouda deps use any of our test deps, try to use the versions we want
-AK_PIP_CONTRAINTS="--constraint $CHPL_HOME/third-party/chpl-venv/test-requirements.txt"
-if ! python3 -m pip install --force-reinstall --upgrade --no-cache-dir $AK_PIP_CONTRAINTS -e .[dev] --user ; then
-  log_fatal_error "installing arkouda"
-fi
-
-# Check installation
-test_start "make check"
-if make check ; then
-  log_success "make check output"
-else
-  log_fatal_error "running make check"
-fi
-test_end
-
-# Run Python unit tests
-test_start "make test-python"
-if make test-python ; then
-  log_success "make test-python output"
-else
-  log_error "running make test-python"
-fi
-test_end
-
-# Run benchmarks
-if [ "${CHPL_TEST_ARKOUDA_PERF}" = "true" ] ; then
-  benchmark_opts="--gen-graphs --dat-dir $CHPL_TEST_PERF_DIR --graph-dir $CHPL_TEST_PERF_DIR/html"
-
-  benchmark_opts="${benchmark_opts} --annotations $CHPL_HOME/test/ANNOTATIONS.yaml"
-  if [[ -n $CHPL_TEST_PERF_DESCRIPTION ]]; then
-    benchmark_opts="${benchmark_opts} --description $CHPL_TEST_PERF_DESCRIPTION"
-  fi
-  if [[ -n $CHPL_TEST_PERF_CONFIG_NAME ]]; then
-    benchmark_opts="${benchmark_opts} --platform $CHPL_TEST_PERF_CONFIG_NAME"
-  fi
-  if [[ -n $CHPL_TEST_NUM_TRIALS ]]; then
-    benchmark_opts="${benchmark_opts} --numtrials $CHPL_TEST_NUM_TRIALS"
-  fi
-  if [[ -n $CHPL_TEST_PERF_START_DATE ]]; then
-    benchmark_opts="${benchmark_opts} --start-date $CHPL_TEST_PERF_START_DATE"
-  fi
-  if [[ -n $CHPL_TEST_PERF_CONFIGS ]]; then
-    benchmark_opts="${benchmark_opts} --configs $CHPL_TEST_PERF_CONFIGS"
+if [ ${CHPL_TEST_ARKOUDA_STOP_AFTER_BUILD} = "false" ]; then
```

Comment:
Just a nit, not important, but I might just set this in the GPU script and then check if it's unset, rather than setting it to false if unset here, but really doing the same thing.

Reply:
I guess my concern is if someone does see `export CHPL_TEST_ARKOUDA_STOP_AFTER_BUILD="true"` in the gpu cron script it'd be easy enough for them to assume that `export CHPL_TEST_ARKOUDA_STOP_AFTER_BUILD="false"` would turn the feature off (which it wouldn't if I updated that check to be "if this variable is defined").

I could update the test to say something to the effect of: if "unset or if false" and then get rid of the defaulting it to false if unset thing at the top of the script.

Reply:
Alright, fine by me.